bigquery/setup.py

```diff
@@ -52,14 +52,15 @@
 
 REQUIREMENTS = [
     'google-cloud-core >= 0.28.0, < 0.29dev',
+    'google-api-core >= 0.1.1, < 0.2.0dev',
```

Comment:
@jonparrott This is what you had in mind, adding an explicit dep on `google-api-core`?